tools/codegen/codespec/api_to_provider_spec_mapper.go

```diff
@@ -328,17 +344,19 @@ func getAPISpecResource(spec *high.Document, resourceConfig *config.Resource, na
 		errResult = errors.Join(errResult, fmt.Errorf("unable to extract '%s.delete' operation: %w", name, err))
 	}
 
-	commonParameters, err := extractCommonParameters(spec.Paths, resourceConfig.Read.Path)
+	commonParameters, err := getCommonParameters(spec.Paths, resourceConfig)
```

Comment:
following on Aastha comment, we could simply preserve calling `extractCommonParameters`?

Reply:
Thanks! Looking a bit deeper I see `extractCommonParameters` is actually useless, as it is populating `CommonParameters` which is not used at all. If you confirm the same feel free to do the clean up of the function and property altogether.